Nodejs/Product/TestAdapterImpl/TestMethodResolver.cs

```diff
@@ -31,6 +31,9 @@ internal class TestMethodResolver : ITestMethodResolver
 
         public string GetCurrentTest(string filePath, int line, int lineCharOffset)
         {
+            // TODO: finish implementing this method
```

Comment:
Is this pending for the PR? If not, shouldn't we remove the unreachable code after the `return null`?

Reply:
Nope, not pending for this PR. I'm happy to remove it; I didn't initially because it looked like someone had started working on it. But I agree now that it should be removed.

Reply:
Actually, I didn't realize how old that unfinished code was. I'm _definitely_ removing it.

Reply:
I've removed the unreachable code as suggested, as well as some more unused code.
I've tried to strike a balance in deleting unused code here; technically, this class does nothing, so we could delete the whole thing. But in case someone _did_ want to implement this, I left the barebones structure. Let me know if you think I should keep ripping stuff out.

Reply:
Alright, I've ripped the whole thing out.